src/Symfony/Component/Finder/Finder.php

```diff
@@ -85,6 +85,18 @@ public function __construct()
     }
 
     /**
+     * Get lookup directories
+     *
+     * @return array List of directories
+     *
+     * @api
+     */
+    public static function getDirs()
```

Comment:
static while it access a property of the object ? this is invalid PHP code (and you would have seen it if it was covered by a test)